logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/CallProfile.kt

```diff
@@ -0,0 +1,16 @@
+package com.wire.kalium.logic.data.call
+
+import com.wire.kalium.logic.feature.call.Call
+import com.wire.kalium.logic.feature.call.CallStatus
+
+data class CallProfile(
+    val calls: Map<String, Call>
+) {
+
+    private val callsList get() = calls.values
+
+    operator fun get(conversationId: String): Call? = calls[conversationId]
+
+    fun incomingCalls(): List<Call> = callsList
+        .filter { it.status == CallStatus.INCOMING }
```

Comment:
This is not being used yet, will be properly used after refactoring `_calls` to `_callProfile` in `CallManagerImpl` and other functions such as `ongoingCalls`, `endedCalls` and etc will also be added.